bazel/test/python_test_repo/tools/bazel

```diff
@@ -0,0 +1 @@
+../../../../tools/bazel
```

Comment:
That's a very tricky question that we haven't hammered out yet. At the moment, our efforts are basically just focused on supporting a single version at a time, which is defined by `tools/bazel`. Because of the constant backwards-incompatible changes, it's difficult to do anything else. Once Bazel has a reasonable degree of API stability, we can start the conversation about a supported compatibility range.